www/include/monitoring/objectDetails/serviceDetails.php

```diff
@@ -533,8 +533,8 @@
 
         $optionsURL = "host_name=" . urlencode($host_name) . "&service_description=" . urlencode($svc_description);
 
-        $query = "SELECT id FROM `index_data` WHERE host_name = '" . $pearDBO->escape($host_name) .
-            "' AND service_description = '" . $pearDBO->escape($svc_description) . "' LIMIT 1";
+        $query = "SELECT id FROM `index_data`,`metrics` WHERE host_name = '" . $pearDBO->escape($host_name) .
```

Comment:
```suggestion
        $query = "SELECT id FROM `index_data`, `metrics` WHERE host_name = '" . $pearDBO->escape($host_name) .
```

Reply:
Thx for your review @cgagnaire, just corrected 👍